### ISSUE TITLE
Stuck in tilled dirt

### ISSUE THREAD

**Issue by ItzCal** [StratusNetwork/Issues#19]
If you jump in the corner of tilled dirt you get stuck in the dirt
https://youtu.be/NzrCsnPhBJs

**Comment by amactus** [comment]
is this a PGM-specific issue?

**Comment by Palmidence** [comment]
@ItzCal ^

**Comment by ItzCal** [comment]
idk, haven't seen it happen on any other server

**Comment by Pablete1234** [comment]
This is a minecraft bug that mojang introduced when they changed soil's collision box to be 0.8 blocks tall instead of 1 block (used to be visually 0.8, but have a full block collision box), the issue happens when you jump ontop of the soil, your feet is at 0.8 and the soil turns into dirt being 1.0 blocks tall, so you get stuck inside it.
The only fix to this would be to teleport the player ontop of the block if they turn soil into dirt (and don't quote me on this but i think they do this, but they don't do it if you jump in the corner)

TLDR: tell mojang about it, not us
